nibabel/filebasedimages.py

```diff
@@ -538,7 +540,30 @@ class SerializableImage(FileBasedImage):
     """
 
     @classmethod
-    def from_bytes(klass, bytestring):
+    def _filemap_from_iobase(klass, ioobject: io.IOBase):
+        """For single-file image types, make a file map with the correct key"""
+        if len(klass.files_types) > 1:
+            raise NotImplementedError(
+                "(de)serialization is undefined for multi-file images"
+            )
+        return klass.make_file_map({klass.files_types[0][0]: ioobject})
+
+    @classmethod
+    def _from_iobase(klass, ioobject: io.IOBase):
```

Comment:
Any reason not to expose this as something `from_stream`?  (and `to_stream`)?

Reply:
Not that I can think of.